packages/expo/src/Notifications/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from './Notifications';
+export {
+  UserInteraction,
+  OnUserInteractionListener,
+  OnForegroundNotificationListener,
+  LocalNotification
+} from './Notifications.types';
```

Comment:
Is there any specific reason for not exporting everything form `./Notifications.types`?
```suggestion
export * from './Notifications.types';
```